### DISCUSSION TITLE
Ergo block found not showing in wallet

### DISCUSSION THREAD

**Discussion by MiningCryptoLive** [coinfoundry/miningcore#1274] [Q&A]
Our pool found another ergo block but it is not showing up in the pool wallet! Any ideas as to why?

**Comment by oliverw** [comment]
Is it visible on the blockchain explorer?

**Reply by ei8ht187** [reply]
ERGO needs 720 confirmations before the reward is in your wallet. This equals to 24h. I had the best experience when I install Nautilus Wallet in the browser and import the wallet with the seedphrase. 
Second best ist with the ERGO native panel, although slow sometimes. 

And what @oliverw said.

**Comment by babelfish2001** [comment]
In the node api swagger, go down to mining and then click on /mining/rewardAddress. Then hit execute and you'll see a reward address pop up that should start with "88" this is a system address that will show any mined ergo transactions while it is still in the 720 confirmations window.

That way you can see the reward before it actually hits your wallet.

**Comment by honorkeles** [comment]
did you guys figure out to unlock the block without waiting for 720 confirmations? I see other pools offers it so there gotta be a way.

**Reply by oliverw** [reply]
> did you guys figure out to unlock the block without waiting for 720 confirmations? I see other pools offers it so there gotta be a way.

This is not a technical issue. You can certainly lower the number of required confirmations for Ergo using Miningcore's `config.json`. Just keep in mind that you need to pre-fund your pool with several block rewards in advance to do this.

**Reply by honorkeles** [reply]
Thank you Oliver

**Comment by MiningCryptoLive** [comment]
Yes it's visible. Hasn't been 720 blocks yet. I thought unlocking early
made it available but you still have to wait for 720 blocks. Should be
good. Thanks
